assets/src/edit-story/panels/size/useSize.js

```diff
@@ -0,0 +1,59 @@
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback, useState, useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+
+function useSize( width, height ) {
+	const [ lockRatio, setLockRatio ] = useState( true );
+	const [ state, setState ] = useState( { width, height } );
+
+	useEffect( () => {
+		setState( { width, height } );
+	}, [ width, height, setState ] );
```

Comment:
`setState` should not be needed as a dependency here. Does eslint require it?

Reply:
Yes, eslint error

Reply:
If I remove it (and similar for `setLockState` in the `useCallback` for the toggle) I get no eslint errors. The `react-hooks/exhaustive-deps` explicitly excludes `setFoo` from a `useState` to be included as a dependency (it can of course only check that as long as it's in the same function and not some argument).

Can you check? It doesn't really matter, as it's promised to be a static function that will never change, so no re-rendering will be done, but it does make the code unnecessarily complex.

Reply:
you're correct, no errors.  maybe my eyes are bad! 😉